### DISCUSSION TITLE
how to generate magnon in text?

### DISCUSSION THREAD

**Discussion by steven-tj23** [mailhexu/TB2J#29] [Q&A]
Can we get the txt file rather than a pic for the magnon spectrum? How?

And how can we specify the q-path as we want? Rather than just using the special points specified in ASE website. For instance, K and K' for MoS2 structure. ASE only define a K point.

**Comment by mailhexu** [comment]
Hi, 
Since this could be helpful for many users, I will implement the functionality to save the magnon data and the specification of arbitrary k-point in the next TB2J version. 
Best regards,
HeXu

**Reply by steven-tj23** [reply]
That would be great! Thanks.
